src/wizard2.cc

```diff
@@ -1694,11 +1693,6 @@ void do_cmd_debug()
 		gain_random_corruption();
 		break;
 
-		/* Create a trap */
-	case 'R':
```

Comment:
Should probably be removed from the manual. (Just blank out the text, don't bother moving items around.)